src/restore/async_batch_restore_service.py

```diff
@@ -32,10 +31,12 @@ def __run_copy_job_for_each(self, restore_items, restoration_job_id):
 
             source_table_reference = restore_item.source_table_reference
             target_table_reference = restore_item.target_table_reference
+
+            self.restore_workspace_creator.create_workspace(
```

Comment:
What will happen if this method will raise an exception?

Reply:
If creating workspace will raise an exception, the whole process of restoration will return error. But I think this is what we want.

Reply:
Yes, but restore item status should be updated as well